### ISSUE TITLE
Error after "nx g @nrwl/workspace:convert-to-nx-project --all"

### ISSUE THREAD

**Issue by mtrefzer** [etkachev/nx-webstorm#78]
## Current Behavior
I converted my nx-project (angular) `nx g @nrwl/workspace:convert-to-nx-project --all` . 
Now I'm unable to select any commands from the plugin. Always getting the following error.

```
java.lang.NullPointerException
	at com.github.etkachev.nxwebstorm.services.MyProjectService.getProjects(MyProjectService.kt:162)
	at com.github.etkachev.nxwebstorm.services.MyProjectService.getProjectList(MyProjectService.kt:84)
	at com.github.etkachev.nxwebstorm.utils.GenerateFormControl.<init>(GenerateFormControl.kt:20)
	at com.github.etkachev.nxwebstorm.ui.RunSchematicPanel.<init>(RunSchematicPanel.kt:42)
	at com.github.etkachev.nxwebstorm.actionlisteners.SchematicSelectionTabListener.runSelectedLogic(SchematicSelectionTabListener.kt:139)
	at com.github.etkachev.nxwebstorm.actionlisteners.SchematicSelectionTabListener.valueChanged$lambda-2(SchematicSelectionTabListener.kt:125)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:885)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:754)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:441)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:825)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:440)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:794)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:492)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04.3
 - Webstorm version 2021.2.3

**Comment by mtrefzer** [comment]
After the conversion of the nx-project there is no longer an entry ´projects´ in the `nx.json` file.
Instead it  has been moved to the `angular.json` file.

**Comment by etkachev** [comment]
Yup. This is same as issue mentioned [here](https://github.com/etkachev/nx-webstorm/issues/76). I have a fix with latest [release](https://github.com/etkachev/nx-webstorm/releases/tag/v0.9.1) that was published to Jetbrains. Might be a few days before the update shows up in the marketplace. Will let you know when it's available. Thanks!

**Comment by etkachev** [comment]
This should be fixed in latest 0.9.1 version. Let me know if you have any other questions. Thanks!
